task/tkn-bundle-oci-ta/0.1/tkn-bundle-oci-ta.yaml

```diff
@@ -28,6 +28,10 @@ spec:
       description: The Trusted Artifact URI pointing to the artifact with
         the application source code.
       type: string
+    - name: SOURCE_CODE_DIR
+      description: Source code directory in the working dir
+      type: string
+      default: /source
```

Comment:
as a user, I would expect to input `source` over `/source`. Making this change would also require a change in the recipe to append the `/`.

Reply:
I could use the recipe to append the `/` but it will only append it to the `oci-ta` task and not the non ta task.
However I updated the parameter to be `source` and added the `/` in the environment variable for both versions of the task.